src/main/java/org/folio/rest/impl/PostOrderLineHelper.java

```diff
@@ -533,6 +561,9 @@ private JsonObject buildInstanceRecordJsonObject(PoLine compPOL, Map<String, Str
   Response buildErrorResponse(int code, String message) {
     final Response result;
     switch (code) {
+      case LIMIT_INTERNAL_HTTP_CODE:
```

Comment:
I realized this wasn't introduced in this PR, and might not even be code that you wrote, but the strategy of defining and passing along these codes seems a little kludgy to me.  Shouldn't we be creating Exception classes, e.g. "InvalidLimitException", or "LimitExceededException", and then just passing the Exception to the buildErrorResponse method?